src/compiler/crystal/codegen/target.cr

```diff
@@ -58,6 +58,17 @@ class Crystal::Codegen::Target
     end
   end
 
+  def size_bit_width
+    case @architecture
+    when "aarch64", "x86_64"
+      64
+    when "arm", "i386", "wasm32"
+      32
+    else
+      raise "BUG: unknown Target#size_bit_width for #{@architecture} target architecture"
```

Comment:
**praise:** Raising here is great to avoid unexpected suprises. We should do the same in `#pointer_bit_width` (in a separate PR).

Reply:
Yup, this is in the AVR PR that changes the method :)